docs/guide/configuration.md

```diff
@@ -53,7 +55,52 @@ And now you'll be able to write queries like this:
 const query = myDb.rawQuery(sql`SELECT * FROM users`);
 ```
 
-## 2. A basic example of migrations folder
+## Example 2: Multiple databases connected to your apps
+
+Connect using multiple databases and credentials used by multiple apps
+
+::: tip DEMO
+Check out [@ts-safeql-demos/basic](https://github.com/ts-safeql/safeql/tree/main/demos/basic) for a working example.
+:::
+
+```json
+// .eslintrc.json
+{
+  // ...
+  "rules": {
+    // ...
+    "@ts-safeql/check-sql": [
+      "error",
+      {
+        "connections": [
+          {
+            // The URL of the database:
+            "databaseUrl": "postgres://postgres:postgres@localhost:5432/my_database_1",
+            // The name of the variable that holds the connection:
+            "name": "myDb1",
+            // An array of operators that wraps the raw query:
+            "operators": ["rawQuery"]
+          },
+          {
+            // The URL of the database:
+            "databaseUrl": "postgres://postgres:postgres@localhost:5432/my_database_2",
+            // The name of the variable that holds the connection:
+            "name": "myDb2",
+            // An array of operators that wraps the raw query:
+            "operators": ["query"]
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+## Example 3: Migrations
+
+If your project contains `.sql` migration files, configuring [`connections.migrationsDir` option](https://safeql.dev/api/index.html#connections-migrationsdir) instead of `databaseUrl` will automatically synchronize the changes in your migrations to a separate "shadow database", which will also be used to retrieve type information related to your queries.
```

Comment:
Omit the domain from the link. It should be relative so it will navigate appropriately both in dev and prod environments.

Reply:
f8d469a9633463984a2444bbe49fa0f61c402059